.github/workflows/osx/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -l
+set -x
+brew install wget coreutils xz pybind11
+wget --no-verbose https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/clang+llvm-11.0.0-x86_64-apple-darwin.tar.xz
+ls
+tar -xJf clang+llvm-11.0.0-x86_64-apple-darwin.tar.xz
+export PATH=$PATH:clang+llvm-11.0.0-x86_64-apple-darwin/bin
+
+cmake CMakeLists.txt
+make
```

Comment:
This command will build whole set of executable from LLVM sources, but we only need Binder plus headers so maybe we can only build ` binder tools/clang/lib/Headers/clang-headers` targets? (this will be ~10x faster)

Reply:
Hi @lyskov ,

no, that uses the CMakeLists.txt from the top directory. It will build only the binder.

The CI works in the same way as on Linux.